docs/rfcs/rfc_0001_deductions_from_the_mission_statement.md

```diff
@@ -0,0 +1,111 @@
+# Deductions from the mission statement
+
+by: Crazy Techno Opossum
+
+This document is a Request For Comments. It is open for criticism and review.
+We expect review to improve the quality of this document.
+
+This document extracts some deductions and technical constraints from the mission statement.
+This document does not define a system that maps those constraints to a particular implementation.
+
+This document's intended audience is people that want to help build the system.
+Some cryptography and mathematical knowledge is expected.
+
+
+## Summary
+
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
+"OPTIONAL" in this document are to be interpreted as described in
+[RFC 2119](https://www.rfc-editor.org/rfc/rfc2119).
+
+
+## Key points from the mission statement
+
+> Imagine a unique social project that takes a web server and turns it into a multiplayer hub for 
+> a purposeful cause. The goal? To create a space where people come together, enjoy games or 
+> activities, and contribute to a shared fund. 
+> This fund isn’t just any fund – it’s a resource that automatically gets divided among the users of the server. 
+> So, the more you engage and participate, the more you receive from the fund. 
+> This innovative project not only brings people closer through fun and interaction but also 
+> enables them to collectively impact their community by distributing funds where they’re needed most.
+
+Technology wise, we extract the following points:
+- A web server.
+- Games and activities.
+- People can contribute to a shared fund.
+- The fund gets distributed among the users of the server.
+- A user's income from the fund is a function of their engagement.
+- Income is also distributed based on necessity.
+
+
+## A study of flows
+
+One key ambiguity in the mission statement is if value is expected to be internal or external to the server.
+We will study the internal case and study the external case separately
+
+### Internal flows of value
+
+A key item of the mission statement is the existence of a shared fund.
+This fund MAY receive value, and that value MUST flow to each user in the server eventually.
+
+Value flow from the shared fund to each user MUST be calculated according to a function.
+This function SHOULD take into account:
+- how much a given user has engaged with the server.
+- how much a given user needs value.
+- global engagement and global need of value.
```

Comment:
This is sub modules, Core function is only distribution, bc that is not a function which throws any questions.
How to distribute, on the other hand is, where we make different approaches and test what works in practice, tho this is a manifest of theoretical nature , thereby i reconstitute,

CORE = Share value
HOW = {engagement, luck, equally}

So in Troubleshooting we go 

CORE(enagement)
CORE(luck)
...

to define methodology.

Following, "need" is very mercurial to define, as, need is a composite and compound vector with immeasurable block size

Capitalism is taking this approach, already, we on the other hand are distributing the money first, before asking for need, as everyone is entitled to value which is needed to thrive without gatekeeping of education or government corruption ^~^

Reply:
See previous thread.

Distributing factoring need is in the mission statement. We can change that mission statement if we disagree with it.

Reply:
do you have a suggestion for the rewording ?

i would like your input on what you would like the mission statement to be like, its not holy scripture yet

we shouldn't multiply complexity without necessity tho, occhams razor

that is our limiting trigger for distribution function sliding scales, its a self readjusting, and refining design rule, to distill a possibility space down to actuality